app/src/main/assets/licenses.html

```diff
@@ -540,6 +540,252 @@ <h2 class="inline">9. Accepting Warranty or Additional Liability.</h2>
     </div>
     <div class="library">
       <!-- https://opensource.org/licenses/Apache-2.0 -->
+      <h1 class="title">DeployGate SDK</h1>
+      <p class="notice">Copyright &copy; DeployGate, tnj, henteko. All rights reserved.</p>
```

Comment:
In older PR, deploygate was added to license.yml but generateLicensePage task was not executed.